Servers/utils/assessment.utils.ts

```
@@ -1,60 +1,91 @@
   );
-  return result.rows.length ? result.rows[0] : null;
+  return result.length > 0;
 };
 
 export const deleteAssessmentByIdQuery = async (
   id: number
-): Promise<Assessment | null> => {
-  const result = await pool.query(
-    `DELETE FROM assessments WHERE id = $1 RETURNING *`,
-    [id]
+): Promise<Boolean> => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)